src/hera.cpp

```diff
@@ -64,6 +64,7 @@ struct hera_instance : evmc_instance {
   hera_wasm_engine wasm_engine = hera_wasm_engine::binaryen;
   hera_evm_mode evm_mode = hera_evm_mode::reject;
   bool metering = false;
+  vector<pair<evmc_address, string>> contract_preload_list;
```

Comment:
I think this should be `vector<pair<evmc_address, vector<uint8_t>>>` so that the at runtime it doesn't need to be loaded every time.